proto/ibc/channel/channel.proto

```diff
@@ -173,3 +195,18 @@ message Packet {
   // block timestamp (in nanoseconds) after which the packet times out
   uint64 timeout_timestamp = 8 [(gogoproto.moretags) = "yaml:\"timeout_timestamp\""];
 }
+
+// PacketAckCommitment defines the genesis type necessary to retrieve and store
+// acknowlegements.
+message PacketAckCommitment {
+  option (gogoproto.goproto_getters) = false;
```

Comment:
are these options needed, I would be wary of adding them, its still unclear what will happen with gogoproro